static/app/views/organizationStats/usageChart/utils.spec.tsx

```diff
@@ -13,7 +13,7 @@ describe('getDateFromMoment', () => {
   // Ensure date remains in UTC
   it('shows the date if interval is >= 24h', () => {
     expect(getDateFromMoment(start)).toBe('Jul 9');
-    expect(getDateFromMoment(start, '7d')).toBe('Jul 9');
+    expect(getDateFromMoment(start, '2d')).toBe('Jul 9');
```

Comment:
```suggestion
    expect(getDateFromMoment(start, '7d')).toBe('Jul 9');
```
this should be kept the same, right?

Reply:
I changed it since 7d is not a valid IntervalPeriod.

Reply:
That is Cathy's suggested change. I changed it from `7d` to `2d`.

Reply:
mb 👍

Reply:
oh i see there's an `IntervalPeriod` object